app/main.py

```diff
@@ -1,3 +1,110 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", float, int]) -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: Union[Distance, float, int]) -> Distance:
```

Reply:
why? thank you

Reply:
It all has fallen, 0 tests passes

Reply:
If you add `from __future__ import annotations` it will work